src/components/Todo/Todo.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from '../User/User';
+
+const Todo = ({ title, completed, user }) => {
+  const status = (completed)
+    ? 'done'
+    : 'not completed yet';
+
+  return (
+    <>
+      <h2>
+        {title}
+      </h2>
+      <p>
+        {`Status: ${status}`}
+      </p>
+      <User {...user} />
```

Comment:
if you need only name then pass only name

Reply:
+1. Change it.